proctor-common/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '8.1.1'
```

Comment:
I actually don't think we should use shadowing at all anymore. The main reason we did this originally was because we were trying to exclude javax api jars from published libraries, but our approach has changed since that point.

Now, we allow javax libs in published jars, but clean them up at the consuming end when needed.

Keeping shadowing and migrating it to gradle is likely to cause us more accidental issues than just getting rid of it at this point.

Reply:
The shadow Jar relocation is widely used when I looked on sourcegraph. May required a bigger discussion/effort for removing the shadow jar.

Reply:
Will create ticket to deprecate this feature as I agree this creates unwanted complexity, but believe this is a larger undertaking for the future